hack/ci/check.sh

```diff
@@ -18,9 +18,86 @@
 
 set -ex
 
-./kubectl-direct_csi info
-./kubectl-direct_csi drives list
-./kubectl-direct_csi drives list | grep -q Available
-./kubectl-direct_csi drives format --all
-sleep 5
-./kubectl-direct_csi drives list
+args=("$@")
+
+function setup_lvm() {
+    sudo truncate --size=1G disk-{1..4}.img
+    for disk in disk-{1..4}.img; do sudo losetup --find $disk; done
+    devices=( $(for disk in disk-{1..4}.img; do sudo losetup --noheadings --output NAME --associated $disk; done) )
+    sudo pvcreate "${devices[@]}"
+    vgname="test-vg-$RANDOM"
+    sudo vgcreate "$vgname" "${devices[@]}"
+    for i in {1..4}; do sudo lvcreate --size=800MiB "$vgname"; done
+}
+
+function install_directcsi()
+{
+    ./kubectl-direct_csi install --image "direct-csi:${args[0]}"
```

Comment:
assign respective args to name in the beginning of the script like
```bash
IMAGE_TAG="$1"
```
and use it here like
```bash
--image "direct-csi:${IMAGE_TAG}"
```